docs/orleans/host/configuration-guide/client-configuration.md

```diff
@@ -57,15 +100,42 @@ Here we set two things:
 
 ## Clustering provider
 
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="orleans-7-0"
+<!-- markdownlint-enable MD044 -->
+
+```csharp
+.UseAzureStorageClustering(
+    options => options.ConfigureTableServiceClient(connectionString);
+```
+
+:::zone-end
+
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="orleans-3-x"
+<!-- markdownlint-enable MD044 -->
+
 ```csharp
 .UseAzureStorageClustering(
     options => options.ConnectionString = connectionString)
 ```
 
+:::zone-end
+
 The client will discover all gateway available in the cluster using this provider. Several providers are available, here in this sample we use the Azure Table provider.
 
 For more information, see [Server configuration](server-configuration.md).
 
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="orleans-7-0"
+<!-- markdownlint-enable MD044 -->
+
+:::zone-end
```

Comment:
Just pointing this out just in case it was a mistake, but this pivot is empty? Do you even need to include it, if it's empty?